src/screens/points/contexts/PointsProfileContext.tsx

```diff
@@ -140,7 +140,7 @@ export const PointsProfileProvider = ({ children }: { children: React.ReactNode
         Alert.alert(i18n.t(i18n.l.points.console.generic_alert));
         throw new RainbowError('Points: Error loading wallet');
       }
-      const signatureResponse = await signPersonalMessage(challenge, wallet, provider);
+      const signatureResponse = await signPersonalMessage(challenge, provider, wallet);
```

Comment:
had to switch the order of the arguments here as `signPersonalMessage` no longer has an optional `provider` but still can have an optional `wallet` arg